### PR TITLE
r-rmumps

### PR DESCRIPTION
**Pull request by sgsokol**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from an official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by sgsokol** [comment]
@conda-forge-admin, please ping team
A recipe r-rmumps passes the tests on linux and win but fails on osx. The failure is due to the absence of libgfortran.3.dylib:
```
2019-12-19T16:56:54.5224900Z dyld: Library not loaded: @rpath/libgfortran.3.dylib
2019-12-19T16:56:54.5228830Z   Referenced from: /usr/local/miniconda/conda-bld/r-rmumps_1576774474264/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/R/lib/libRblas.dylib
2019-12-19T16:56:54.5231390Z   Reason: image not found
```
This library is referenced in build.sh originally written by skeleton cran and then modified by me to fix an if-else logic which is not relevent for this discussion.
So, my question is: if libgfortran.3.dylib is no more available (is it v4 now?) how can I get an updated build.sh with the current version of libgfortran for osx?
Thanks,
Serguei Sokol.

**Comment by sgsokol** [comment]
@conda-forge/help-r error in windows build due to conda's bad file name patter.

Resticting r-rmumps package to `rbase >=3.6`, I've got an error on win_64:
2019-12-23T14:06:27.4914832Z OSError: [WinError 123] The filename, directory name, or volume label syntax is incorrect: 'C:\\Miniconda\\conda-bld\\r-rmumps_1577109046063\\work' -> 'C:\\Miniconda\\conda-bld\\r-rmumps_1577109046063\\work_moved_r-rmumps-5.2.1_7-r>=36h3ba9a2d_0_win-64_main_build_loop'

I suppose that the intended part of name was `...r-rmumps-5.2.1_7-r36h3ba9a2d_0_win-64...` not `...r-rmumps-5.2.1_7-r>=36h3ba9a2d_0_win-64...` (extra `>=`). It's probabe that the presense of the `>` in the name that makes OS choke.

Please fixe.

**Comment by sgsokol** [comment]
@conda-forge/help-r please review

**Comment by npavlovikj** [comment]
@sgsokol , the recipe LGTM. However, I would like to hear from @conda-forge/help-r regarding the OSX + r-base 3.5 issue before merging it. That issue has been occurring with other PRs as well (https://github.com/conda-forge/staged-recipes/pull/10297), and I am not sure limiting the R version for OSX is a solution to it.

**Comment by sgsokol** [comment]
@conda-forge/help-r
After r-base-3.5.1 rebuild, it works also on osx. Could someone merge, please?

File: recipes/r-rmumps/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+test:
+  commands:
+    - $R -e "library('rmumps')"           # [not win]
+    - "\"%R%\" -e \"library('rmumps')\""  # [win]
+
+about:
+  home: http://mumps.enseeiht.fr/, https://github.com/sgsokol/rmumps/
+  license: GPL-2
+  summary: 'Some basic features of MUMPS (Multifrontal Massively Parallel sparse direct Solver)
```
Comment (npavlovikj) [inline recipes/r-rmumps/meta.yaml]: This summary is too long. Please rename this summary to `description`, and put something shorter for summary, e.g.:
```
summary: Multifrontal Massively Parallel sparse direct Solver
```

Comment (sgsokol) [reply recipes/r-rmumps/meta.yaml]: Done. BTW, thanks for a prompt review.

**Comment by npavlovikj** [comment]
Thanks for the prompt fixes and contributing, @sgsokol 🎉

**Pull request merged**